src/l500/l500-color.cpp

```diff
@@ -493,10 +493,44 @@ namespace librealsense
         return tags;
     }
 
-    std::vector<uint8_t> l500_color::get_raw_intrinsics_table() const
+    ivcam2::intrinsic_rgb l500_color::get_intrinsics_table() const
```

Comment:
I don't like this function name. If it actually reads the table from FW, let's call it `read_intrinsics_table`

Reply:
Changed